site/gdocs/utils.ts

```diff
@@ -85,13 +89,13 @@ export const useLinkedDocument = (
         errorMessage = `Article with slug "${linkedDocument.slug}" isn't published.`
     }
 
-    //todo replace with getCanonicalUrl
-    const subdirectory =
-        linkedDocument.type === OwidGdocType.DataInsight ? "data-insights/" : ""
     return {
         linkedDocument: {
             ...linkedDocument,
-            slug: `${subdirectory}${linkedDocument.slug}${queryString}${hash}`,
```

Comment:
I really think we'll miss query strings and hashes sooner rather than later, for topic page key insights and linking to the subscribe section on our homepage.

I guess as a workaround we have the absolute URL method, but can I make the case for keeping them around and just filtering out the `tab` param from gdocs?

Reply:
It's unintuitive for the users to edit the gdoc URL and add our own hash/query params. I hope we'll find a better solution for that. And so far we didn't really use the current affordance, so let's cross the bridge when we get there.